similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientSimilarityViewFactory.java

```diff
@@ -313,48 +232,12 @@ private void setDescendantsMap(VocabularyTerm root, Map<VocabularyTerm, Collecti
 
         this.logger.info("Normalizing term frequency distribution...");
         // Normalize all the term frequencies to be a proper distribution
+        double maxFreq = termFreq.get(HP_ROOT);
```

Comment:
This crashes the Component. `termFreq` is a `Map<VocabularyTerm, Double>`, and `HP_ROOT` is a String, so `.get` returns null.